mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
> Issuing a move order that fails (e.g. into water)

Wouldn't it make more sense to generally ignore invalid move orders completely and simply continue with the last valid order?